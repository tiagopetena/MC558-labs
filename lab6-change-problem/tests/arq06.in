75 9102
25 606 2
26 175 1
57 769 3
31 230 2
92 655 3
17 107 2
86 839 3
14 457 3
39 205 3
69 20 2
99 243 2
59 568 2
2 707 1
81 911 2
37 671 1
80 696 3
28 383 3
10 973 3
84 260 3
55 920 3
27 563 3
11 781 2
35 303 2
38 210 1
9 770 1
49 654 1
51 949 2
7 755 2
47 653 1
73 798 1
24 662 1
16 189 2
78 846 2
66 960 1
85 538 1
48 135 1
83 864 2
13 532 3
42 95 2
43 467 1
3 986 2
30 790 3
65 642 2
33 977 2
63 685 2
68 565 2
19 170 2
18 912 1
23 584 1
88 769 2
93 227 2
94 977 2
8 484 2
62 394 1
12 393 3
90 320 3
67 981 2
98 753 3
64 827 3
58 225 2
100 407 3
82 415 3
32 669 2
74 498 1
61 295 3
89 36 2
21 357 2
52 824 3
29 540 3
60 181 1
1 174 3
79 499 2
72 24 1
53 770 1
96 423 2
