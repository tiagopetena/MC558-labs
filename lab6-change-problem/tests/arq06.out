7881 84011
